gen/AArch64Compile_gen.ml

```diff
@@ -1705,11 +1715,10 @@ module Make(Cfg:Config) : XXXCompile_gen.S =
               Some rB,init,cs@lift_code [ldr_mixed rB rA MachSize.S128 0; gctype rB rB],st
           | R,Some (CapaSeal,Some _) -> assert false
           | R,Some (Neon n,None) ->
-              let c = sxtw r2 rd in
               let r3,st = next_reg st in
-              let cs0,st = calc0_gen csel st V64 r3 r2 in
-              let rB,init,cs,st = LDN.emit_load_idx n st p init loc r3 in
-              Some rB,init,Instruction c::pseudo cs0@cs,st
+              let cs0,st = calc0_gen csel st vdep r3 r2 in
```

Comment:
This has a bug of lost dependency, we do not need to call `calc0_gen` here and simply do
```
let rB,init,cs,st = LDN.emit_load_idx n vdep st p init loc r2 in
Some rB,init,pseudo cs0@cs,st
```